src/site/styles/_our-community.scss

```diff
@@ -0,0 +1,34 @@
+@import "vars";
+
+.our-community {
```

Comment:
A lot of these styles are duplicated from the home and about styles... not sure if we want to move them to a better shared location

Reply:
Sure, we can revisit that after this is merged in

Reply:
created #12